e2e/cypress/support/index.js

```diff
@@ -15,6 +15,84 @@
 
 // Import commands.js using ES2015 syntax:
 import './commands'
+import { TEST_USER, TEST_USER2, TEST_ADMIN } from '../common'
 
-// Alternatively you can use CommonJS syntax:
-// require('./commands')
+const postLogin = (user) => {
+  const { username, password } = user
+  return cy.request({
+    url: '/api/login',
+    method: 'POST',
+    body: {
+      username,
+      password
+    }
+  })
+}
+
+const loginAsUser = (user) => {
+  postLogin(user).then((res) => {
+    const userData = res.body
+    window.localStorage.setItem('loggedInUser', JSON.stringify(userData))
+  })
+}
+
+Cypress.Commands.add('loginAsUnregisteredUser', () => {
```

Comment:
Logging in via API instead of via UI was inspired by https://www.youtube.com/watch?v=5XQOK0v_YRE